build.gradle

```diff
@@ -48,6 +48,12 @@ repositories {
 
 // In this section you declare the dependencies for your production and test code
 dependencies {
+    // Spring DI
```

Comment:
We should get away from doing multiple // comments. Describe this in one big block at the top